funfact/initializers.py

```diff
@@ -125,3 +125,22 @@ def __call__(self, shape):
         shape = as_tuple(shape)
         std = (self.scale / shape[self.axis])**0.5
         return std * self.distribution(shape)
+
+
+def vectorize_initializer(initializer, post: bool = True):
+    '''Vectorizes an initializer.
+
+    Args:
+        post (bool):
+            If True, the last index of shape is considered the vectorizing
+            index. If False, the first index of shape tuple is considered
+            the vectorizing index.
+    '''
+    if initializer is None:
```

Comment:
I understand that this is to handle the case when an abstract tensor has `None` as the initializer.

However, I think it might be semantically more straightforward if this is handled at the terminal level (AbstractTensor), since vectorizing None is not a very meaningful action.

What we can do is to simply do nothing in `AbstractTensor.vectorize` if the initializer is None, because in that case it'll default to `Normal`, which does not need vectorization.

Reply:
Done.